cmd/provision/legacy.go

```diff
@@ -201,13 +201,13 @@ func (p *provision) createLegacyCredential(printf shared.FormatFn) (*keySecret,
 		return nil, err
 	}
 
-	resp, err := p.ApigeeClient.Do(req, nil)
+	_, err = p.ApigeeClient.Do(req, nil)
 	if err != nil {
 		return nil, err
 	}
-	if resp.StatusCode > 299 {
-		return nil, fmt.Errorf("creating credential, status: %d", resp.StatusCode)
-	}
+	// if resp.StatusCode > 299 {
```

Comment:
If we don’t need this, we should remove it.

Reply:
Sure I will remove it. ApigeeClient explicitly checks the status code and return error on non-2XX codes.